<?xml version="1.0"?>
<launch>

    <arg name="cmd_vel_topic" default="/cmd_vel_prio_low" />
    <arg name="input_pose_topic" default="pcl_closest_obj/output_pose" />
    <arg name="camera_link" default="d435_arm_camera_depth_optical_frame" />

    <node pkg="mir_3d_image_segmentation" type="closed_loop_pick_base_controller"
        name="closed_loop_pick_base_controller" output="screen">
        <remap from="~input_pose" to="$(arg input_pose_topic)" />
        <remap from="~cmd_vel" to="$(arg cmd_vel_topic)" />
        <remap from="~closest_obj_event_in" to="/pcl_closest_obj/event_in" />
        <param name="camera_link" value="$(arg camera_link)" type="str" />
    </node>

    <include file="$(find mir_3d_image_segmentation)/ros/launch/closest_obj.launch" />

</launch>

